posts/2017-04-12-serverless-v1.11.0.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Separate Package and Deploy command, CloudWatch Log event support with Serverless v1.11
```

Comment:
or "New in Serverless v.1.11: Trigger functions with via cloudwatch infrastructure events"